packages/@cdktf/provider-generator/lib/get/__tests__/provider.test.ts

```diff
@@ -14,3 +14,5 @@ getProvider(
 );
 
 getProvider(new TerraformProviderConstraint("oci@= 4.13.0"));
+
+getProvider(new TerraformProviderConstraint("DataDog/datadog@~> 3.0"));
```

Comment:
The other version constraints are pinned, I guess that would make sense for this provider as well